scalameta/dialects/shared/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -142,7 +142,7 @@ package object dialects {
     .withAllowStarWildcardImport(true)
     .withAllowProcedureSyntax(false)
     .withAllowDoWhile(false)
-    .withAllowPlusMinusUnderscoreAsPlaceholder(true)
+    .withAllowUnderscoreAsTypePlaceholder(true)
```

Comment:
Should we also change it for `Scala212Source3` and `Scala213Source3` ?

Reply:
for scala2 we use `allowPlusMinusUnderscoreAsIdent`